### DISCUSSION TITLE
LGTM数のContribution数への寄与について

### DISCUSSION THREAD

**Discussion by daikisuyama** [increments/qiita-discussions#26] [要望・アイデア・不具合]
より多くの人が良いと感じた記事がQiitaへの貢献度として評価されるのは当然なので、記事のLGTM数がContribution数に寄与するという点は尤もだと思います。

ただ、同じLGTMであってもQiitaへの貢献度は異なると思うので、view数や流入元などを元に傾斜をつけてContribution数に加算する等についてはお考えになっているのでしょうか。

**Comment by kskwtnk** [comment]
@daikisuyama 

ご意見ありがとうございます！

> view数や流入元などを元に傾斜をつけてContribution数に加算する等についてはお考えになっているのでしょうか。

こちらについては「考えはしたものの実施に至らなかった」が回答となります

最近Contribution計算を新しくしたのですが、その際に計算式について色々と議論が出ました

- 古い記事と新しい記事は一緒で良いのか
- 1行の編集リクエストと100行の編集リクエストは同じで良いのか

などなど、色々な観点で話をしていました
view数も議論にあがったのですが

- viewが多い方が良いとも、少ない方が良いとも言いづらい
    - バズって多くの人に見られて役に立った記事→view数が多く、価値が高いはず
    - 明らかに間違った内容で炎上して多くの人に見られた記事→view数は多いが、情報としての価値は低いはず
    - あまりにも玄人向けだけど、その分野の人にはとても役立つ記事→view数は少ないが、価値が高いはず

こういった具合に、一概に計算式として出せるものではないという結論にいたり、今のような比較的シンプルな式に落ち着きました
神の視点から見るともっと良い算出方法がある可能性はありますが、システムとして提供する計算式……という意味では、傾斜をつけないという考えでいます！

**Reply by daikisuyama** [reply]
ご丁寧にご説明ありがとうございます。
システム化は確かに難しいと感じました。
自分も何か良いものを思いついたら投稿させていただきます。
